index.html

```diff
@@ -12,18 +12,18 @@
     <main id="main">
         <div id="first">
```

Comment:
You did an awesome job in this section, but it would be great if you tried using Semantic HTML elements in the header section
example: <header></header>

Reply:
- [ ] I suggest to give the reason of your changes as it will allow the student to learn, since giving code review aim also to share knowledge :recycle: